eventyay_business/templates/eventyay_business/subscriptions/list.html

```
@@ -0,0 +1,57 @@
+                    <tr>
+                        <td>{{ sub.organizer.name }}</td>
+                        <td>{{ sub.tier_version }}</td>
+                        <td>
+                            <span class="badge badge-{% if sub.status == 'active' %}success{% elif sub.status == 'canceled' %}danger{% else %}warning{% endif %}">
+                                {{ sub.get_status_display }}
+                            </span>
+                        </td>
+                        <td>{{ sub.starts_at|date:"SHORT_DATE_FORMAT" }}</td>
+                        <td>{{ sub.ends_at|date:"SHORT_DATE_FORMAT"|default:"-" }}</td>
+                        <td class="text-right">
+                            <a href="{% url "plugins:eventyay_business:subscriptions.edit" pk=sub.pk %}" class="btn btn-default btn-sm">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)